drivers/hmis/app/graphql/mutations/submit_form.rb

```diff
@@ -82,6 +82,8 @@ def resolve(input:)
           record = Hmis::Hud::HmisService.find_by(owner: record.owner) # Refresh from View
         elsif record.is_a? HmisExternalApis::AcHmis::ReferralRequest
           HmisExternalApis::AcHmis::CreateReferralRequestJob.perform_now(record)
+        elsif record.is_a? Hmis::Hud::Enrollment
+          record.save_in_progress
```

Comment:
I wonder if eventually (or soon-ish) we should roll this into methods on the record.  That way instead of a case statement here, we could just call `record.handle_form_submit` or something similar.  It could probably wrap `perform_side_effects` as well.

Reply:
Yeah that might be good. Will consider when doing https://www.pivotaltracker.com/story/show/185505416